inframanager/api_handler/api_handler.go

```diff
@@ -333,17 +360,23 @@ func insertRule(log *log.Entry, ctx context.Context, p4RtC *client.Client, macAd
 	}
 
 	logger.Infof("Inserting entry into the cni tables")
-	if err = p4.InsertCniRules(ctx, p4RtC, macAddr, ipAddr, portID, ifaceType); err != nil {
+	ep, err = p4.InsertCniRules(ctx, p4RtC, ep, ifaceType)
```

Comment:
The function implementation is different for MEV and DPDK. Need to handle both signatures or merge the implementation in cni*.go

Reply:
Leaving this comment for Venkatest to address as separate commit on this branch.

Reply:
Looks like its taken care.